nibabel/xmlutils.py

```diff
@@ -27,7 +27,11 @@ def _to_xml_element(self):
     def to_xml(self, enc='utf-8'):
         """ Output should be an xml string with the given encoding.
         (default: utf-8)"""
-        return tostring(self._to_xml_element(), enc)
+        ele = self._to_xml_element()
+        if ele is None:
```

Comment:
My personal preference is for the explicit None check, if that is the 'got nothing' marker - because it makes it more explicit what you're expecting the variable to be and why you are doing the check.